### PR TITLE
[Windows] Add PHP to the toolset

### PR DESCRIPTION
**Pull request by V-Zabayrachny**
# Description
This pull request moves the PHP version to the toolset. Also added a check in a pester tests to make we are installing the correct version with accordance to the toolset.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by KVROHITH098** [comment]
I will do these,

Please assign these to me thank you.

**Comment by KVROHITH098** [comment]
I am working on this. I will update u soon, 

I need a Runbook like a step BY step guide to help me? Can u make one?

File: images/win/scripts/Tests/PHP.Tests.ps1 [hunk 1]
```diff
@@ -1,13 +1,15 @@
 Describe "PHP" {
-    It "Check <ToolName> in the PATH" -TestCases @(
-        @{ ToolName = "php" }
-        @{ ToolName = "composer" }
-    ) {
-        "$ToolName --version" | Should -ReturnZeroExitCode
+    It "Check PHP version" {
+        $phpMajorMinor = (Get-ToolsetContent).php.version
+        "php --version" | Should -BeLike "*${phpMajorMinor}*"
```
Comment (miketimofeev) [inline images/win/scripts/Tests/PHP.Tests.ps1:4]:
```suggestion
        php --version | Should -BeLike "*${phpMajorMinor}*"
```

File: images/win/scripts/Tests/PHP.Tests.ps1 [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+    }
+
+    It "Check Composer in the PATH" {
+        "composer --version" | Should -ReturnZeroExitCode
```
Comment (miketimofeev) [inline images/win/scripts/Tests/PHP.Tests.ps1:8]:
```suggestion
        composer --version | Should -ReturnZeroExitCode
```

File: images/win/scripts/Tests/PHP.Tests.ps1 (hunk 1 above)
Comment (V-Zabayrachny) [reply images/win/scripts/Tests/PHP.Tests.ps1]: Thanks, I fixed it.

File: images/win/scripts/Tests/PHP.Tests.ps1 (hunk 2 above)
Comment (V-Zabayrachny) [reply images/win/scripts/Tests/PHP.Tests.ps1]: Thanks, I fixed it.

**Review by miketimofeev — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Pull request merged**